packages/itwin/grouping-mapping-widget/src/components/Mappings/hooks/useFetchExtractionStates.ts

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { useExtractionClient } from "../../context/ExtractionClientContext";
+import { ExtractorState } from "@itwin/insights-client";
+import { useMutation, useQuery } from "@tanstack/react-query";
+import type { GroupingMappingApiConfig } from "../../context/GroupingApiConfigContext";
+import type { ExtractionRequestMapping, ExtractionRunRequest, Mapping } from "@itwin/insights-client";
+import { useState } from "react";
+import { ExtractionStates } from "../Extraction/ExtractionStatus";
+import { STATUS_CHECK_INTERVAL } from "../../Constants";
+
+export const useFetchExtractionStates = ({
+  iModelId,
+  getAccessToken,
+}: GroupingMappingApiConfig) => {
+
```

Comment:
Do not add empty lines at the beginning or the end of a block of code.

Reply:
I have made this change.